doc/002-local-dev.md

```diff
@@ -1,184 +1,207 @@
+
+Subsequent `ziti` CLI commands will automatically re-use this session token. You'll need to perform this login step again when the token expires.
 
-### Starting  Routers
+### Enroll the Routers
 
-The Ziti Fabric requires at least one router (fabric router or edge router). There are four predefined configuration files 
-for running routers in `etc/` named `001.yml` to `004.yml`. 
+The Ziti Fabric requires at least one router to function. There are four predefined configuration files
+for running routers in `etc/` named `001.yml` to `004.yml`.
 
 Each configuration file refers to certificate and private keys kept in `etc/ca/intermediate/certs` and
-`etc/ca/intermediate/private/`. The steps for starting the router is to first register the router then 
-start it.
+`etc/ca/intermediate/private/`.
 
 Register:
 
 ```bash
-ziti-fabric create router etc/ca/intermediate/certs/XXX-client.cert.pem
+ziti fabric create router etc/ca/intermediate/certs/SERIAL-client.cert.pem
 ```
 
-Where `XXX` is replaced with `001` through `004`.
+Where `SERIAL` is replaced with `001` through `004`.
+
+### Start the Routers
 
-Run:
+You'll need at least one running edge router. Remember to initialize the environment in each shell.
 
 ```bash
-ziti-router run etc/XXX.yml
+ziti-router run etc/SERIAL.yml
 ```
 
-Where `XXX` is replaced with `001` through `004`.
+Where `SERIAL` is replaced with `001` through `004`.
 
-### Starting Router As An Edge Router
-
-Edge routers are routers that have the Edge functionality enabled and allow Ziti SDK enabled application to connect to 
-Ziti. Starting an edge eouter requires that the controller be running with the Edge functionality
+Edge routers are routers that have the Edge functionality enabled and allow Ziti SDK enabled application to connect to
+Ziti. Starting an edge router requires that the controller be running with the Edge functionality
 enabled. This requires the use of the `ctrl.with.edge.yml` configuration to run the controller (see example above).
 
 To start an edge router the Edge REST API will be used to prime the enrollment process and the
 `ziti-router enroll` command will be used to finalize the process. The enrollment command will handle adding the fabric
-and edge router entries necessary. Using the `ziti-fabric create router` command should not and can not be run.
-
+and edge router entries necessary. Using the `ziti fabric create router` command should not and can not be run.
 
-There is only one example configuration file for an edge router: 
+There is only one example configuration file for an edge router:
 `etc/edge.router.yml`. Additional configuration files can be created by copying and altering the
 file. Specifically the identity section needs to point to unique file locations that do not collide with other identity
-file and the listening ports need to not be in use.
+files, and the listening ports need to be available to bind.
 
-### Authenticate w/ the Ziti Edge API
+## Ziti Edge REST API Primer
```

Comment:
i don't want you to put it there, but this really belongs in the edge repo imo. not here.  Alternaltively you can just remove this and all the rest from here down since its' all about using ziti not dev'ing on ziti. it doesn't belong